node/get-started/overview.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Overview'
+description: 'Understanding Nodes in the Atoma Network'
+---
+
+# What is a Node?
+
+A node is a computer that joins the Atoma Network to help process AI tasks. Think of nodes as workers in a distributed workforce - each one contributes its computing power to run AI models and gets paid for their work.
+
+When you run a node, you're essentially renting out your computer's processing power to users who need to run AI models but don't have the hardware themselves.
+
+## Types of Nodes
+
+There are two main ways your node can process AI tasks:
+
+### 1. Sampling Consensus Nodes
+These are the most common type of nodes. They:
+- Run AI models in a standard way
+- Work together with other nodes to verify that results are correct
+- Are perfect for running public AI models where privacy isn't a concern
+- Participate in our unique "Sampling Consensus" system where multiple nodes check each other's work
+
+### 2. Confidential Compute Nodes
+These are specialized nodes for private, sensitive AI tasks. They:
+- Use special hardware security features called "Trusted Execution Environments" (TEEs)
+- Process data in a completely private, encrypted space
+- Are ideal for sensitive data that needs extra privacy protection
+- Cost more to run but provide enhanced security guarantees
+
+## How to Join the Network
+
+Getting your node up and running involves a few simple steps:
+
+1. **Choose Your Role**: Decide if you want to run a standard Sampling Consensus node or a Confidential Compute node
+
+2. **Subscribe to Tasks**: 
+   - Browse available AI tasks on the network
+   - Choose which tasks you want to process
+   - Set your price (how much you want to be paid)
+   - Specify how much computing power you can offer
+
+3. **Set Your Parameters**:
+   - Pick the specific task IDs you want to work on
+   - Set your price per unit of computation
+   - Define your computing power limits
+   - Register your node with a unique identifier
+
+## How Nodes Make Money
+
+Once your node is running:
+1. The network automatically assigns you tasks
+2. Your node processes the AI workloads
+3. You submit proof that you completed the work
+4. You get paid in tokens for successful computations
+
+The more reliable your node is, the better reputation you build, leading to more task assignments and better earning potential.
```

Comment:
not tasks, the term we use is `stack`. But we can say something like `the better reputation you build, more often is your node picked to process AI workloads and therefore higher chances to earn value`